docs/installation/certificate.asciidoc

```diff
@@ -0,0 +1,355 @@
+
+image::certificate/9-Radius-key.png[scaledwidth="100%",alt="Radius key"]
+
+5. Import or open your certification authority certificate file (.crt) with a text editor, then copy and paste the key into the "RADIUS Server Certification Authority Certificate" field.
+
+image::certificate/10-Radius-CA.png[scaledwidth="100%",alt="Radius CA"]
+
+6. Turn on the "Find RADIUS Server intermediate CA(s) automatically" and "Validate certificate chain" option.
+
+image::certificate/13-Radius-intermediate-chain.png[scaledwidth="100%",alt="Radius chain"]
+
+7. Restart all `radiusd` services that are running, including `radius-auth`, `radiusd-load-balancer`, `radiusd-acct`, `radiusd-eduroam`, and `radiusd-cli`. Restart them one server at a time. On the web admin page, go to `Status > Services`.
+
+image::certificate/11-Services.png[scaledwidth="100%",alt="Services"]
+
+Alternatively, you can use the following commands in the command-line interface (CLI):
+
+[source, shell]
+----
+systemctl restart packetfence-haproxy-admin
+systemctl restart packetfence-haproxy-portal
+bin/pfcmd service radiusd restart
+----
+
+=== 5. Management of SSL/TLS Certificate for RADIUS
+
+==== 5.1 Information about Renewal of SSL/TLS Certificate
+
+To renew the SSL/TLS certificate for RADIUS, you can use your previous Certificate Signing Request (CSR) or generate a new one with the exact same information. You can find all the necessary information in `System Configuration > SSL Certificate > View RADIUS Certificate`. It's important not to use a wildcard certificate for RADIUS.
+
+image::certificate/12-Radius-status.png[scaledwidth="100%",alt="Radius status"]
+
+==== 5.2 Renewal of SSL/TLS Certificate
+
+To renew the SSL/TLS certificate for RADIUS, follow these steps:
+
+1. Open the PacketFence web admin interface `System Configuration > SSL Certificate > RADIUS > Edit RADIUS Certificates`.
+
+2. Import the new certificate file (.crt) or paste the content of the new certificate using a text editor into the "RADIUS Server Certificate" field.
+
+image::certificate/8-Radius-certificate.png[scaledwidth="100%",alt="Radius certificate"]
+
+3. Add the new Certification Authority certificate.
+
+image::certificate/10-Radius-CA.png[scaledwidth="100%",alt="Radius CA"]
+
+4. Turn on the "Find RADIUS Server intermediate CA(s) automatically" and "Validate certificate chain" option.
+
+image::certificate/13-Radius-intermediate-chain.png[scaledwidth="100%",alt="Radius chain"]
+
+5. Press "Save" to finish the renewal.
+
+6. Restart all `radiusd` services that are running, including `radius-auth`, `radiusd-load-balancer`, `radiusd-acct`, `radiusd-eduroam`, and `radiusd-cli`. Restart them one server at a time. On the web admin page, go to Status > Services.
+
+image::certificate/11-Services.png[scaledwidth="100%",alt="Services"]
+
+Alternatively, you can use the following commands in the command-line interface (CLI):
+
+[source, shell]
+----
+systemctl restart packetfence-haproxy-admin
```

Comment:
Why restart haproxy-admin and haproxy-portal if we are renewing the RADIUS certificate?